### PR TITLE
Adding xmlvalidator

### PR DESCRIPTION
**Pull request by mjw99**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mdd31** [comment]
Comment that I'm happy to be included as a maintainer.

**Comment by mjw99** [comment]
@conda-forge/help-python This PR is ready to be reviewed.

File: recipes/xmlvalidator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+build:
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (dopplershift) [inline recipes/xmlvalidator/meta.yaml:21]: I don't see any C code in this package? If not, then can remove the entire `build` section and put `noarch: python` under the top-level `build`.

**Comment by mjw99** [comment]
@dopplershift Many thanks for the suggestions. I have made changes accordingly.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**